.github/workflows/Pytest.yml

```diff
@@ -1,12 +1,4 @@
-on:
-  push:
-    branches:
-      - main
-      - master
-  pull_request:
-    branches:
-      - main
-      - master
+on: push
```

Comment:
What is this change for?

Reply:
```
on:
  push:
    branches:
      - main
      - master
      - "bacpop-*"
  pull_request:
    branches:
      - main
      - master
      - "bacpop-*"
```

I set this pattern up in mcstate2 and it's working well (https://github.com/mrc-ide/mcstate2/commit/d267d0e2cff3d1aa2cc04903966d7b16f0078396)